flat/svelte.mjs

```diff
@@ -0,0 +1,43 @@
+import rules from './frontend-rules.mjs';
+import globals from 'globals';
+import jsdoc from 'eslint-plugin-jsdoc';
+import js from '@eslint/js';
+import babelConfig from './babel.mjs';
+import jestConfig from './jest.mjs';
+import vitestConfig from './vitest.mjs';
+import es from 'eslint-plugin-es';
+import eslintPluginSvelte from 'eslint-plugin-svelte';
+
+/** @type { import("eslint").Linter.Config[] } */
+const configs = {};
+
+configs['flat/svelte-base'] = [
+    {
+        name: 'tao:ignore-folders',
+        ignores: ['public', 'dist', 'coverage']
+    },
```

Comment:
What do you think if we would _only_ use the .eslintignore file in consumer projects? For me, it gives a little more control - that file's contents are almost certain to vary a bit from one project to the next, so why even try to extend from centralised defaults?

Reply:
Sounds good to me. I was doing this mainly to reduce the amount of customization in the consumer projects. If they forget (or just don't know how) to add these folders to .eslintignore, the linting process will take much longer. Also, adding them as the first config, could improve efficiency, as they will be ignored for the rest of the linting process. 

I'm fine with passing this responsability to consumers. Lets see if anyone else has something to say about this.

Reply:
I'm also for using the `.eslintignore` files in each consumer project. Internal structure may differ from one project to another.